parsec/backend/cli/run.py

```diff
@@ -316,25 +335,30 @@ def run_cmd(
 
     with cli_exception_handler(debug):
 
+        if ssl_certfile or ssl_keyfile:
+            ssl_context = ssl.create_default_context(ssl.Purpose.CLIENT_AUTH)
+            if ssl_certfile:
+                ssl_context.load_cert_chain(ssl_certfile, ssl_keyfile)
+            else:
+                ssl_context.load_default_certs()
+        else:
+            ssl_context = None
+
         config = BackendConfig(
             administration_token=administration_token,
             db_url=db,
             db_drop_deleted_data=db_drop_deleted_data,
             db_min_connections=db_min_connections,
             db_max_connections=db_max_connections,
             blockstore_config=blockstore,
+            ssl_enabled=ssl_context is not None,
```

Comment:
This won't work given the SSL can be handled by a third party between the client and the backend (this is what is done when host on Heroku, where the client create an SSL connexion with the Heroku datacenter, which internally forward a non-ssl connection to the backend)

The solution is instead to provide a `--backend-addr`/`PARSEC_BACKEND_ADDR` parameter in the configuration.